src/main/java/Deadline.java

```diff
@@ -0,0 +1,15 @@
+public class Deadline extends Task {
+    public static final char TASK_TYPE = 'D';
+    private final String BY;
```

Comment:
variable names must in camelCase which in this case "private final String by;" would be appropriate